### DISCUSSION TITLE
IndexError: list index out of range

### DISCUSSION THREAD

**Discussion by Rayen023** [obss/sahi#903] [Q&A] [answered]
Hey Guys need some help figuring out how to solve this error 

Performing inference on images:   7%|▋         | 21/322 [00:23<05:35,  1.11s/it]
File "/gpfs/scratch/rayen/sahi/sahi/predict.py", line 602, in predict
    object_prediction_gt = ObjectPrediction.from_coco_annotation_dict(
  File "/gpfs/scratch/rayen/sahi/sahi/annotation.py", line 470, in from_coco_annotation_dict
    return cls.from_coco_bbox(
  File "/gpfs/scratch/rayen/sahi/sahi/annotation.py", line 425, in from_coco_bbox
    xmin = bbox[0]
IndexError: list index out of range
loading annotations into memory...
Done (t=0.00s)
creating index...
index created!

**My code :** 

```
from sahi.utils.yolov8 import (
    download_yolov8s_model,
)

from sahi import AutoDetectionModel
from sahi.utils.cv import read_image
from sahi.utils.file import download_from_url
from sahi.predict import get_prediction, get_sliced_prediction, predict
from IPython.display import Image

yolov8_model_path = "models/woodv8l.pt"

model_type = "yolov8"
model_path = yolov8_model_path
model_device = "cuda:0" # or 'cuda:0'
model_confidence_threshold = 0.25

slice_height = 512
slice_width = 512
overlap_height_ratio = 0.2
overlap_width_ratio = 0.2

source_image_dir = "wood_val/"

predict(
    model_type=model_type,
    model_path=model_path,
    model_device=model_device,
    model_confidence_threshold=model_confidence_threshold,
    source=source_image_dir,
    slice_height=slice_height,
    slice_width=slice_width,
    overlap_height_ratio=overlap_height_ratio,
    overlap_width_ratio=overlap_width_ratio,
    dataset_json_path = "wood_val/val_data.json",
)

```

Note : I am using Version: 0.11.14 with a custom dataset which I ve been able to train Yolov8 with and get expected results, so data structure should be fine. sahi also seems to work fine on a couple first images.

**Comment by sanjeevnara7** [comment, marked as answer]
I get the same issue when running the CLI commands with a pretrained YOLOv8 model. I suspect that the COCO json annotations may be corrupt (which could have happened when converting from YOLO to COCO format in my case), but would like to know if any others have thoughts

**Reply by Rayen023** [reply]
Yes, solved it my recreating the COCO json file
